app.js

```diff
@@ -370,7 +370,9 @@ function showProfile(req, res) {
 		} else {
 			res.render('pages/profile', {
 				title: 'Profile',
-				profileData: result.profileData
+        profileData: result.profileData,
+        countryView: result.countries,
+        preferences: result.preferences,
```

Comment:
Let's indent this a little neater. If this only happens on github you can ignore this.

Reply:
Yes I think this is just a small github thing. In my app.js the changed data is aligned like the removed line on 373.